src/ast.hpp

```diff
@@ -218,7 +218,7 @@ namespace Sass {
   public:
     Vectorized(size_t s = 0) : hash_(0)
     { elements_.reserve(s); }
-    Vectorized(const std::vector<T> vec) :
+    Vectorized(const std::vector<T>& vec) :
```

Comment:
Passing by value is better if you're going to copy anyway, because that allows for a copy-free constructor if the argument at call site is an rvalue. For example:

```c++
Vectorized a({1, 2, 3}); // No copies made, only 2 moves.
```

Reply:
OK, thanks for the heads up. Possibly that we have
dozens of other places where this was not known.